packages/kie-bc-editors/src/jslienzo/JsLienzo.ts

```diff
@@ -14,21 +14,68 @@
  * limitations under the License.
  */
 
+/**
+ * Javascript API that allows interactions with nodes and connectors
+ * in Kogito editors.
+ */
 export interface JsLienzo {
```

Comment:
`JsLienzoEnvelopeApi` can have just a header doc like `JsLienzo` and reference the jsdoc of methods here